pyomo/scripting/runtests.py

```diff
@@ -31,136 +31,136 @@ def runPyomoTests(argv=None):
-
-    if _options.docdir:
-        docdir = os.path.abspath(_options.docdir)
-        if not os.path.exists(docdir):
-            raise ValueError("Invalid documentation directory, "
-                             "path does not exist")
-    elif _options.doctests:
-        docdir = os.path.join('pyomo','doc','OnlineDocs')
-    else:
-        docdir = None
-
-    if _options.dir is None:
-        # the /src directory (for development installations)
-        os.chdir( os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))) )
-    else:
-        if os.path.exists(_options.dir):
-            os.chdir( _options.dir )
-
-    print("Running tests in directory %s" % os.getcwd())
-    _options.cat = os.environ.get('PYUTILIB_UNITTEST_CATEGORY', _options.cat)
-    if _options.cat == 'all':
-        if 'PYUTILIB_UNITTEST_CATEGORY' in os.environ:
-            del os.environ['PYUTILIB_UNITTEST_CATEGORY']
-    elif _options.cat:
-        os.environ['PYUTILIB_UNITTEST_CATEGORY'] = _options.cat
-        print(" ... for test category: %s" % os.environ['PYUTILIB_UNITTEST_CATEGORY'])
-
-    options=[]
-    if _options.coverage:
-        options.append('--coverage')
-    if _options.verbose:
-        options.append('-v')
-    if _options.doctests:
-        options.append('--with-doctest')
-    if outfile:
-        options.append('-o')
-        options.append(outfile)
-
-    if len(args) > 1:
-        mydirs = args[1:]
-    else:
-        mydirs = [os.path.join('pyomo','pyomo'), 'pyomo-model-libraries']
-    #
-    dirs=[]
-    for dir in mydirs:
-        if dir in _options.exclude:
-            continue
-        if dir.startswith('-'):
-            options.append(dir)
-        if dir.startswith('pyomo'):
-            if os.path.exists(dir):
-                dirs.append(dir)
-            elif '.' in dir:
-                dirs.append(os.path.join('pyomo','pyomo',dir.split('.')[1]))
-        else:
-            if os.path.exists('pyomo.'+dir):
-                dirs.append('pyomo.'+dir)
-            else:
-                dirs.append(os.path.join('pyomo','pyomo',dir))
-    #
-    excluding = set()
-    for e in _options.exclude:
-        excluding.add(os.path.join('pyomo','pyomo',e))
-    testdirs = []
-    for topdir in dirs:
-        for root, subdirs, files in os.walk(topdir):
-            if not '__init__.py' in files:
-                # Skip directories that do not contain a __init__.py file.
-                continue
-            for f in files:
-                if f.startswith("test"):
-                    skip=False
-                    for e in excluding:
-                        if root.startswith(e):
-                            skip=True
-                    if not skip:
-                        testdirs.append(root)
-                    break
-    #
-    if docdir:
-        testdirs.append(docdir)
-
-    return pyutilib.dev.runtests.run('pyomo', ['runtests']+options+['-p','pyomo']+testdirs)
+# def OLD_runPyomoTests():
```

Comment:
Whoops. I commented it out to test my theory that it wasn't doing anything, but didn't remove it because it had been living there for so long. @jsiirola - Keep the `OLD_runPyomoTests` method or junk it?